### PR TITLE
Feat Cart

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added Cart

### CODE REVIEW

File: Cart/Cart.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ .card__container{
+     margin-top:5rem;
+     grid-area: main;
+     display: flex;
+     flex-direction: column;
+ }
+ 
+ .card__hori2{
+     padding:1rem 4rem ;
+     width:35rem !important;
```
Comment (msdeshmukh009) [inline Cart/Cart.css:24]: Any particular reason to use `!important` ? Maybe try solving the problem using specificity.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

File: Cart/Cart.css (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply Cart/Cart.css:24]: Okay I will try to improve it through specificity

**Comment by nshanbhag20001412** [comment]
For the cost display section ...you might add padding to make it good better. Rest is good too go.

**Review by nshanbhag20001412 — APPROVED** [review]

**Review by Amandeep-sinha — APPROVED** [review]
Try to avoid !important rest looks good....

**Review by Devansu-Yadav — APPROVED** [review]
Looks good to merge. Some suggestions - 

- For the `cart__payment` container, you can add some `padding` appropriately.
- You can also align the text for each billing item within this container.

S-Beauty - Google Chrome 21-02-2022 14_56_18 (2)

**Pull request merged**